fiftyone/core/odm/ontology.py

```
@@ -0,0 +1,64 @@
+        ],
+    }
+
+    name = StringField(required=True)
+    version = IntField(required=True, default=1)
+    type = StringField(required=True, choices=_ONTOLOGY_TYPE_VALUES)
+    description = StringField(default=None)
+    root = DictField()
+    created_at = DateTimeField()
+    last_modified_at = DateTimeField()
+
+    def save(self, *args, **kwargs):
```

Ruff (0.15.7)

[warning] Missing type annotation for `*args`

(ANN002)